ui/page/privacy/account_mixer_page.go

```diff
@@ -68,175 +68,176 @@ func (pg *AccountMixerPage) OnNavigatedTo() {
+										txt := l.Theme.H6(values.String(values.StrMixer))
+										return txt.Layout(gtx)
+									}),
+									layout.Rigid(func(gtx C) D {
+										return layout.Inset{Left: values.MarginPadding20, Right: values.MarginPadding40}.Layout(gtx, func(gtx C) D {
+											return pg.mixerProgress.Layout(gtx)
+										})
+									}),
+								)
+							})
+						}),
+					)
+				})
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layout.Inset{Right: values.MarginPadding15, Left: values.MarginPadding15, Top: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+					return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							ic := l.Theme.Icons.MixedTxIcon
+							return ic.Layout12dp(gtx)
+						}),
+						layout.Rigid(func(gtx C) D {
+							txt := l.Theme.H6(values.String(values.StrMixed))
+							return layout.Inset{Left: values.MarginPadding11}.Layout(gtx, func(gtx C) D {
+								return txt.Layout(gtx)
+							})
+						}),
+						layout.Flexed(1, func(gtx C) D {
+							return layout.Inset{Right: values.MarginPadding25}.Layout(gtx, func(gtx C) D {
+								return layout.E.Layout(gtx, func(gtx C) D {
+									return components.LayoutBalance(gtx, pg.Load, mixedBalance)
 								})
 							})
-					},
-					func(gtx C) D {
-						return pg.mixerSettingsLayout(gtx)
-					},
-					func(gtx C) D {
-						return pg.dangerZoneLayout(gtx)
-					},
-				}
-				return pg.pageContainer.Layout(gtx, len(widgets), func(gtx C, i int) D {
-					m := values.MarginPadding10
-					if i == len(widgets) {
-						m = values.MarginPadding0
-					}
-					return layout.Inset{Bottom: m}.Layout(gtx, widgets[i])
+						}),
+					)
 				})
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layout.Inset{Right: values.MarginPadding40, Left: values.MarginPadding40}.Layout(gtx, func(gtx C) D {
+					return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Max.X = gtx.Constraints.Max.X/2 - 20
+							return layout.W.Layout(gtx, l.Theme.Separator().Layout)
+						}),
+						layout.Rigid(func(gtx C) D {
+							return layout.Inset{Left: values.MarginPadding10, Right: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+								ic := l.Theme.Icons.MixerIcon
+								return layout.Center.Layout(gtx, func(gtx C) D {
+									return ic.Layout36dp(gtx)
+								})
+							})
+						}),
+						layout.Rigid(func(gtx C) D {
+							return layout.E.Layout(gtx, l.Theme.Separator().Layout)
+						}),
+					)
+				})
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layout.Inset{Right: values.MarginPadding15, Left: values.MarginPadding15, Bottom: values.MarginPadding40}.Layout(gtx, func(gtx C) D {
+					return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							ic := l.Theme.Icons.UnmixedTxIcon
+							return ic.Layout12dp(gtx)
```

Comment:
the same line 174

Reply:
This does not apply because the function expects layout.Dimension